frontend/javascripts/oxalis/view/context_menu.tsx

```diff
@@ -1107,6 +1112,45 @@ function ContextMenuInner(propsWithInputRef: Props) {
     maybeViewport,
   } = props;
 
+  const segmentIdAtPosition = globalPosition != null ? getSegmentIdForPosition(globalPosition) : 0;
+  const { visibleSegmentationLayer, volumeTracing } = props;
+  const hasNoFallbackLayer =
+    visibleSegmentationLayer != null &&
+    "fallbackLayer" in visibleSegmentationLayer &&
+    visibleSegmentationLayer.fallbackLayer == null;
+  const [segmentVolume, boundingBoxInfo] = useFetch(
+    async () => {
+      if (contextMenuPosition == null) return [];
+      if (visibleSegmentationLayer != null && volumeTracing != null && hasNoFallbackLayer) {
+        const tracingId = volumeTracing.tracingId;
+        const tracingStoreUrl = Store.getState().tracing.tracingStore.url;
+        const mag = getResolutionInfo(visibleSegmentationLayer.resolutions);
+        const [segmentSize] = await getSegmentVolumes(
+          tracingStoreUrl,
+          tracingId,
+          mag.getHighestResolution(),
```

Comment:
I think (99% sure but please test), `getHighestResolution` returns the coarsest (poorest) resolution. The wording is poor and should be renamed, since "high res" sounds like the best quality, but in fact this refers to the high numbers (e.g., mag 32) which indicates a high degree of downsampling (== information loss).
We should rename this to coarsest and finest resolution, but not in this PR. 
For now, you should use `getLowestResolution` here and in line 1137.

Reply:
this makes a lot more sense. the bounding boxes were off all along, and I didnt test them thoroughly yet. now they look much more realistic upon first view, but I will follow Flo's advice and make sure that they still seem to be correct upon second and third view